docs/api/actions-and-connectors/create.asciidoc

```diff
@@ -64,8 +63,161 @@ For more information, refer to
+(Optional, string) The REST API HTTP request method to create a case comment in
+the third-party system. Valid values are either `patch`, `post`, and `put`. The
+default value is `put`.
+
+`createCommentUrl`::
+(Optional, string) The REST API URL to create a case comment by ID in the
+third-party system. You can use a variable to add the external system ID to the
+URL. If you are using the `xpack.actions.allowedHosts` setting, make sure the
+hostname is added to the allowed hosts. For example:
++
+[source,text]
+----
+https://testing-jira.atlassian.net/rest/api/2/issue/{{{external.system.id}}}/comment
+----
+
+`createIncidentJson`::
+(TBD, string) A JSON payload sent to the create case URL to create a case. You
+can use variables to add case data to the payload. Required variables are
+`case.title` and `case.description`. For example:
++
+[source,json]
+----
+{
+	"fields": {
+	  "summary": {{{case.title}}},
+	  "description": {{{case.description}}},
+	  "labels": {{{case.tags}}}
+	}
+}
+----
++
+NOTE: Due to Mustache template variables (which is the text enclosed in triple
+braces, for example, `{{{case.title}}}`), the JSON is not validated when you
+create the connector. The JSON is validated after the Mustache variables have
+been placed when REST method runs. Manually ensure that the JSON is valid to
+avoid future validation errors; disregard Mustache variables during your review.
+
+`createIncidentMethod`::
+(Optional, string) The REST API HTTP request method to create a case in the
+third-party system. Valid values are `patch`, `post`, and `put`. The default
+value is `post`.
+
+`createIncidentResponseKey`::
+(Required, string) The JSON key in the create case response that contains the
+external case ID.
+
+`createIncidentUrl`::
+(Required, string) The REST API URL to create a case in the third-party system.
+If you are using the `xpack.actions.allowedHosts` setting, make sure the
+hostname is added to the allowed hosts.
+
+`getIncidentResponseExternalTitleKey`::
+(Required, string) The JSON key in get case response that contains the external
+case title.
+
+`getIncidentUrl`::
+(Required, string) The REST API URL to get the case by ID from the third-party
+system. If you are using the `xpack.actions.allowedHosts` setting, make sure the
+hostname is added to the allowed hosts. You can use a variable to add the
+external system ID to the URL. For example:
++
+[source,text]
+----
+https://testing-jira.atlassian.net/rest/api/2/issue/{{{external.system.id}}}
+----
++
+NOTE: Due to Mustache template variables (the text enclosed in triple braces,
+for example, `{{{case.title}}}`), the JSON is not validated when you create the
+connector. The JSON is validated after the Mustache variables have been placed
+when REST method runs. Manually ensure that the JSON is valid, disregarding the
+Mustache variables, so the later validation will pass.
+
+`hasAuth`::
+(TBD, boolean) If true, a username and password for login type authentication
```

Comment:
> (TBD, boolean) If true, a username and password for login type authentication

Likewise, I haven't tested yet whether this is required or optional and if the latter what its default value is.